<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="DESKTOP-6UOC2TJ" failures="2" tests="3" name="petTest" time="8.509" errors="0" timestamp="2023-05-15T21:57:39 IST">
  <testcase classname="api.test.UserTests" name="testPostUser" time="4.248">
    <failure type="java.lang.AssertionError" message="expected [200] but found [405]">
      <![CDATA[java.lang.AssertionError: expected [200] but found [405]
at api.test.UserTests.testPostUser(UserTests.java:40)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1541)
... Removed 34 stack frames]]>
    </failure>
  </testcase> <!-- testPostUser -->
  <testcase classname="api.test.UserTests" name="testGetUserByName" time="2.012"/>
  <testcase classname="api.test.UserTests" name="testDeleteUserByName" time="1.639">
    <failure type="java.lang.AssertionError" message="expected [200] but found [405]">
      <![CDATA[java.lang.AssertionError: expected [200] but found [405]
at api.test.UserTests.testDeleteUserByName(UserTests.java:79)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1541)
... Removed 34 stack frames]]>
    </failure>
  </testcase> <!-- testDeleteUserByName -->
</testsuite> <!-- petTest -->
